src/restorer/mnemonic-restorer.ts

```diff
@@ -87,27 +93,31 @@ export function masterPubKeyRestorerFromEsplora(toRestore: MasterPublicKey) {
 // From state
 
 export interface StateRestorerOpts {
-  lastUsedExternalIndex: number;
-  lastUsedInternalIndex: number;
+  lastUsedExternalIndex?: number;
+  lastUsedInternalIndex?: number;
 }
 
 function restorerFromState<R extends MasterPublicKey>(
```

Comment:
would u add a test for `restorerFromState` and a `StateRestorerOpts` with both members undefined ?